libr/anal/p/anal_lanai_gnu.c

```diff
@@ -50,7 +50,7 @@ static int lanai_op(RAnal *a, RAnalOp *op, ut64 addr, const ut8 *buf, int len, R
 	disasm_obj.symbol_at_address_func = &symbol_at_address;
 	disasm_obj.memory_error_func = &memory_error_func;
 	disasm_obj.print_address_func = &generic_print_address_func;
-	disasm_obj.endian = !a->config->big_endian;
+	disasm_obj.endian = !R_ARCH_CONFIG_IS_BIG_ENDIAN (a->config);
```

Comment:
IS_LITTLE_ENDIAN

Reply:
also maybe we should use shorter macros or names here. also, as long as we support multiple endians, not just two, we should avoid using that as a boolean, if its not big endian, it can be anything, not just little